enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -31,7 +31,11 @@
                         </h2>
                     </div>
                     <span class="text">
-                        ${_("In order to link your account, we need consent to share your course data with {provider}.").format(provider=sso_provider)}
+                        % if not course_id:
+                            ${_("In order to link your account, we need consent to share your course data with {provider}.").format(provider=ec_name)}
```

Comment:
Before your account can be linked, you must agree to share your course data with {provider}.